game_components/entitiies/entity.py

```diff
@@ -0,0 +1,20 @@
+import hashlib
+import time
+
+from abc import ABC, abstractmethod  # abstract classes
+
+
+class Entity(ABC):
+    health = None
```

Comment:
Initializing variables outside of `__init__` isn't required, and I think it's generally better not to do it. You can just do `self.health = health` inside of the `__init__` and it'll have the same effect.